monday/query_joins.py

```diff
@@ -167,6 +166,75 @@ def delete_item_query(item_id):
     return query
 
 
+# COLUMNS RESOURCE QUERIES
+def create_column(
+    board_id: int, column_title: str, column_type: None, defaults: dict[str, any] = None, description: str = None
+):
+    """C```reate a new column by board ID
+
+    Args:
+        board_id [int]: Board ID
+        column_title [str]: Column's title
+        column_type [ColumnTypes]: Column's type
+        defaults [Dict[str,Any]]: Default value
+        description [str]: Column's description
+
+    Return:
+        [Dict[str,Any]]: Column's Monday details after creation
+    """
+    defaults = defaults if defaults else ""
```

Comment:
`defaults` is typed as a dict? Should it be an empty string if not provided?

Reply:
@matiboy you're calling out the correct thing -- I think the reason we're able to get away with `dict` or `str` is because `monday_json_string` converts the dictionary into a string -- I do agree that for correctness, this should be:

```
defaults = defaults or {}
```